### PR TITLE
Add kind to test-runner image.

### PR DESCRIPTION
**Pull request by wlynch**
# Changes

Adds [Kubernetes-in-Docker (kind)](https://kind.sigs.k8s.io/) to test-runner image. This is being
used for e2e tests (see
https://github.com/tektoncd/experimental/tree/master/results/test/e2e).

Blocking https://github.com/tektoncd/experimental/pull/673

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/images/test-runner/Dockerfile [hunk 1]
```diff
@@ -42,6 +42,8 @@ RUN go get -u github.com/golang/dep/cmd/dep
 RUN go get -u github.com/google/licenseclassifier
 RUN go get -u github.com/jstemmer/go-junit-report
 RUN GO111MODULE="on" go get -u github.com/raviqqe/liche
+RUN GO111MODULE="on" go get -u knative.dev/test-infra/tools/githubhelper
```
Comment (vdemeester) [inline tekton/images/test-runner/Dockerfile:45]: Interesting, I though we were not using `githubhelper` anymore, you may be outdated :sweat_smile:

**Comment by wlynch** [comment]
/hold

yeah my base branch was wayyy behind. stay tuned.

**Review by vdemeester — APPROVED** [review]
/cc @afrittoli

**Pull request merged**